src/azul/service/hca_response_v5.py

```diff
@@ -65,6 +65,8 @@ class PaginationObj(JsonObject):
     search_after_uid = StringProperty()
     search_before = DefaultProperty()
     search_before_uid = StringProperty()
+    next = StringProperty()
```

Comment:
@GPelayo this suggestion I will not follow through with because this might affect how the `pagination` response object looks to the user. Which I believe we want to keep it as user friendly as possible, I could be wrong.

Reply:
I see. We can use the `name` argument of `StringProperty`, so the property can be user-friendly without being a reserved word.

```suggestion
    next_ = StringProperty(name='next')
```